File: benchmark/results/java-multilang/run-004/a-mcp/output/catalog/value_objects/ProductDescription.java
package com.metaengine.demo.catalog.value_objects;

import java.time.OffsetDateTime;

// ProductDescription value object.

public class ProductDescription {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public ProductDescription(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}

File: benchmark/results/ts-monolith-opus-inline/run-004/b-baseline/output/src/catalog/value-objects/ProductDescription.ts
import { Id } from '../../shared/value-objects/Id';
import { Timestamp } from '../../shared/value-objects/Timestamp';

/** ProductDescription value object. */
export interface ProductDescription {
  id: Id;
  createdAt: Timestamp;
  updatedAt: Timestamp;
  name: string;
  description: string;
}
